logdna/client.go

```diff
@@ -128,19 +128,19 @@ func MakeRequestView(c *Client, url string, urlsuffix string, method string, pay
 
 // CreateAlert creates a Preset Alert with the provided payload
 func (c *Client) CreateAlert(url string, payload ViewPayload) (string, error) {
-	result, err := MakeRequestAlert(c, url, "/v1/config/alert", "POST", payload)
+	result, err := MakeRequestAlert(c, url, "/v1/config/presetalert", "POST", payload)
```

Comment:
meant to ask - is this name change a bug, or something else? not sure this change is best to be included here either way

Reply:
this name change has to do with complying with a change to the configuration-api of updating the endpoint from alert => presetalert and yeah maybe best to do this in a separate pr